lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
<p>Myslím že lepší by bylo rovnou skočit na <code>pd.DataFrame.from</code><em>records(data).  A pak třeba ukázat jak se z toho dostane <code>street</em>id</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Viz také diskuze výše</p>